src/registrar/admin.py

```diff
@@ -192,6 +205,139 @@ def response_change(self, request, obj):
                     % obj.name,
                 )
             return HttpResponseRedirect(".")
+
+        if GET_SECURITY_EMAIL in request.POST:
+            try:
+                contacts = obj._get_property("contacts")
+                email = None
+                for contact in contacts:
+                    if ["type", "email"] in contact.keys() and contact[
+                        "type"
+                    ] == "security":
```

Comment:
Weird linter thing?

Reply:
@zandercymatics please don't review this yet. I am still making changes that is why this is set to draft. And yes that is for sure black formatting